File: internal/collector/treesitter/testdata/test_kind/java/java-junit5/teardown/TeardownTest.java
// SPDX-License-Identifier: Apache-2.0

package fixture;

import org.junit.jupiter.api.AfterAll;
import org.junit.jupiter.api.AfterEach;

public class TeardownTest {
    @AfterAll
    static void tearDownAll() {}

    @AfterEach
    void tearDown() {}
}

File: internal/collector/treesitter/testdata/test_kind/kotlin/kotlin-junit/teardown/TeardownTest.kt
// SPDX-License-Identifier: Apache-2.0

package fixture

import org.junit.jupiter.api.AfterAll
import org.junit.jupiter.api.AfterEach

class TeardownTest {
    @AfterAll
    fun tearDownAll() {}

    @AfterEach
    fun tearDown() {}
}
